dashboard/test/models/school_test.rb

```diff
@@ -21,4 +21,76 @@ class SchoolTest < ActiveSupport::TestCase
       }
     )
   end
+
+  test 'high needs false when no stats data' do
+    school = School.find_by_id('20000100206')
```

Comment:
Do we need this specific school (by id)? Can we just get the first, or create one on the fly (via FactoryGirl's `create :school`)?

Reply:
sure, updated